src/include/users/widgets.rb

```diff
@@ -1830,17 +1832,9 @@ def InitDefaults(key)
       exp_date = ""
       if expire != "0" && expire != ""
         # 'expire' is expected to be number of days since 1970-01-01
-        out = Convert.to_map(
-          SCR.Execute(
-            path(".target.bash_output"),
-            Ops.add(
-              Builtins.sformat(
-                "date --date='1970-01-01 00:00:01 %1 days' ",
-                expire
-              ),
-              "+\"%Y-%m-%d\""
-            )
-          )
+        out = SCR.Execute(
+          path(".target.bash_output"),
+          "/usr/bin/date --date='1970-01-01 00:00:01 '#{expire.shellescape}' days' +\"%Y-%m-%d\""
```

Comment:
It have to be there, as shellescape have to *not* be enclosed in quotes, otherwise it produce wrong result, so I intentionally end quote and start it again after interpolation.